### PR TITLE
create products table

### PR DESCRIPTION
**Pull request by Kwalsh2379**
Created a products table

### CODE REVIEW

File: db/init_db.js [hunk 1]
```diff
@@ -11,6 +11,16 @@ async function buildTables() {
     // drop tables in correct order
 
     // build tables in correct order
+    await client.query(`
+    CREATE TABLE products(
+      id SERIAL PRIMARY KEY,
+      name varchar(255) UNIQUE NOT NULL,
+      description VARCHAR(255) NOT NULL,
+      price NUMERIC NOT NULL,
+      imageURL VARCHAR(255) NOT NULL DEFAULT null,
```
Comment (albert4770) [inline db/init_db.js:20]: Image url? Where is this gonna be hosted?

Comment (Kwalsh2379) [reply db/init_db.js:20]: This will be the picture of the product

**Review by travispinkerton — APPROVED** [review]
looking good!

**Review by albert4770 — APPROVED** [review]
Table looking good

**Pull request merged**